src/IonSharedSymbolTable.ts

```diff
@@ -31,7 +63,7 @@ export class SharedSymbolTable {
     return this._version;
   }
 
-  get symbols() : string[] {
-    return this._symbols;
+  get numberOfSymbols(): number {
```

Comment:
Do we like this spelling over something like `size`, `length`, or `count`?

Reply:
I prefer length but its purely an opinion